projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -0,0 +1,38 @@
+import { standardUser } from '../../sample-data/shared-users';
+import { generateMail, randomString } from '../user';
+import { clickHamburger } from '../checkout-flow';
+
+export function loginRegisteredUser() {
+  standardUser.registrationData.email = generateMail(randomString(), true);
+  cy.requireLoggedIn(standardUser);
+  cy.reload();
+}
+
+export function clickMyAccountMenuOption(){
+  cy.visit('/');
+  cy.get('#cx-header', { timeout: 10000 }).should('be.visible');
+  cy.onMobile(() => {
+    clickHamburger();
+  });
+  cy.get('.accNavComponent button').click();
+}
+
+export function clickCustomerSupportMenuOption(){
+  cy.get('.accNavComponent li:nth-child(14)').should('contain.text', 'Customer Service');
+  cy.get('.accNavComponent li:nth-child(14) a').click();
+}
+
+export function verifyTicketListingPageVisit(){
+  cy.url().should('include','/my-account/support-tickets');
+  cy.get('cx-customer-ticketing-list').should('exist');
+}
+
+export function clickFirstTicketFromTicketListing(){
+  cy.get('#ticketing-list-table tbody tr:nth-child(1) .cx-ticketing-list-data:nth-child(1) a.cx-ticketing-list-value').should('exist');
+  cy.get('#ticketing-list-table tbody tr:nth-child(1) .cx-ticketing-list-data:nth-child(1) a.cx-ticketing-list-value').click();
```

Comment:
nit:  Here magic number 1 is better because at least the func is making reference to it in the function name.
Nevertheless, it should be better with FIRST_TICKET = 1

Reply:
Done.